src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/policies/.content.xml

```diff
@@ -531,6 +531,84 @@
                     </default>
                 </teaser>
             </commerce>
+#end
+#if ( $includeFormsheadless == "y" )
+            <adaptiveForm jcr:primaryType="nt:unstructured">
+              <formcontainer jcr:primaryType="nt:unstructured">
+                <jcr:content
+                  jcr:primaryType="nt:unstructured"
+                  mergeList="{Boolean}true"/>
+                <default
+                  jcr:description="Policy of the Adaptive Form V2 Container"
```

Comment:
@rismehta  I am a little doubtful here. I have created a new policy that has component group 'apptitle - Adaptive Forms', as of now only formcontainer is part of this group. 
I have used this policy in v2 template and template type.